yt_dlp/extractor/bibeltv.py

```diff
@@ -1,27 +1,204 @@
+            'view_count': data.get('viewCount'),
+            'like_count': data.get('likeCount'),
+        }
+
+    def _real_extract(self, url):
+        crn_id = self._match_id(url)
+        webpage = self._download_webpage(url, crn_id)
+        nextjs_data = self._search_nextjs_data(webpage, crn_id)
+        video_data = try_get(nextjs_data,
+                             lambda x: x['props']['pageProps']['videoPageData']['videos'][0], dict)
+        if not video_data:
+            raise ExtractorError('Missing video data.')
+
+        return self._extract_video(video_data)
+
+
+class BibelTvSerienIE(BibelTVIE):
+    IE_DESC = 'BibelTV'
+    _VALID_URL = r'https?://(?:www\.)?bibeltv\.de/mediathek/serien/(?:crn/)?(?P<id>\d+)'
+    _RETURN_TYPE = 'playlist'
+
+    IE_NAME = 'bibeltv:serien'
+
+    _TESTS = [{
+        'url': 'https://www.bibeltv.de/mediathek/serien/333485-ein-wunder-fuer-jeden-tag',
+        'playlist_mincount': 400,
+        'info_dict': {
+            'id': '333485',
+            'title': 'Ein Wunder für jeden Tag',
+            'description': 'Tägliche Kurzandacht mit Déborah Rosenkranz.',
+        },
+    }]
 
     def _real_extract(self, url):
         crn_id = self._match_id(url)
-        return self.url_result(
-            self.BRIGHTCOVE_URL_TEMPLATE % crn_id, 'BrightcoveNew')
+        webpage = self._download_webpage(url, crn_id)
+        nextjs_data = self._search_nextjs_data(webpage, crn_id)
+        serie_data = try_get(nextjs_data,
+                             lambda x: x['props']['pageProps']['seriePageData'], dict)
+        if not serie_data:
+            raise ExtractorError('Missing series data.')
+
+        video_data = serie_data.get('videos', ())
+
+        return self.playlist_result(
+            LazyList(map(self._extract_url_info, video_data)),
+            playlist_id=crn_id,
+            playlist_title=serie_data.get('title'),
+            playlist_description=clean_html(serie_data.get('description')),
+            playlist_count=len(video_data),
+        )
+
+
+class BibelTvLiveIE(BibelTVIE):
+    _VALID_URL = r'https?://(?:www\.)?bibeltv\.de/livestreams/(?P<id>[a-z]+)'
+    IE_NAME = 'bibeltv:live'
+    _TESTS = [{
+        'url': 'https://www.bibeltv.de/livestreams/bibeltv/',
+        'info_dict': {
+            'id': 'bibeltv',
+            'ext': 'mp4',
+            'title': 're:Bibel TV',
+            'live_status': 'is_live',
+            'thumbnail': 'https://streampreview.bibeltv.de/bibeltv.webp',
+        },
+        'params': {'skip_download': 'm3u8'},
+    }, {
+        'url': 'https://www.bibeltv.de/livestreams/impuls/',
+        'only_matching': True,
+    }]
+
+    def _real_extract(self, url):
+        stream_id = self._match_id(url)
+        webpage = self._download_webpage(url, stream_id)
+        json_btv_data = self._search_json(
+            r'\s*data\s*=', webpage, 'bibeltvData', stream_id,
+            transform_source=lambda x: js_to_json(x.replace('`', "'")))
```

Comment:
We should probably modify js_to_json to support `, but out of scope for this PR

Reply:
After merging master you can just use this now:
```suggestion
            transform_source=js_to_json)
```